ci/prepare-documentation.py

```diff
@@ -121,6 +123,23 @@ def add_page(output_root, parent, *, id=None, path=None, title=None):
         (output_root / image_rel_path.parent).mkdir(parents=True, exist_ok=True)
         shutil.copyfile(OMZ_ROOT / image_rel_path, output_root / image_rel_path)
 
+    non_md_links = [ref.url for ref in page.external_references()
+                    if ref.type == 'link' and not ref.url.endswith('.md')]
+
+    for non_md_link in non_md_links:
+        parsed_non_md_link = urllib.parse.urlparse(non_md_link)
+
+        if parsed_non_md_link.scheme or parsed_non_md_link.netloc:
+            continue # not a relative URL
+
+        if parsed_non_md_link.fragment:
+            continue # link to markdown section
+
+        relative_path = (OMZ_ROOT / Path(path).parent / non_md_link).resolve().relative_to(OMZ_ROOT)
```

Comment:
does it will work if user specify absolute path on the local machine as link?
e.g. something like here (https://github.com/openvinotoolkit/open_model_zoo/blob/e5b37f8eb6c46b2b65a87ce0b3b4ff7d97e51c7d/models/public/mobilenet-yolo-syg/accuracy-check.yml), but in the documentation

Reply:
No, there will be `ValueError: <local path> does not start with <OMZ_ROOT path>`.

Reply:
then ok